package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "node-thrift-test",
+  "version": "0.0.1",
+  "dependencies": {}
+}
```

Comment:
This file should be nested somewhere... was it accidentally created here for the synthetic target that was generate by codegen? If you remove it, is it re-created?

Reply:
Yes, these files were accidentally created, fixed now.